app/controllers/commissions_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CommissionsController < ApplicationController
+  before_action :set_commission, only: [:show, :edit, :update, :destroy]
+
+  def index
```

Comment:
We don't need a lot of these methods. I think that you may have autogenerated these methods, when in reality, we only need the create, show, update, and destroy methods for now.